backend/inspirehep/disambiguation/utils.py

```diff
@@ -109,3 +110,30 @@ def update_author_names(author, signatures):
         author["name"]["name_variants"] = name_variants
     author.update(dict(author))
     return author
+
+
+def normalize_lit_author_name(lit_author_name, author_name):
+    lit_author_name_tokens = re.split(", | ", lit_author_name)
+    lit_author_last_names = lit_author_name.split(",")[0].split(" ")
+    lit_author_first_names_reordered = []
+    lit_author_last_names_reordered = []
+
+    author_last_names = set(author_name.split(",")[0].split(" "))
+    author_first_names = (
+        set(author_name.split(",")[1].split(" ")) if "," in author_name else []
+    )
+
+    for name_idx, name in enumerate(lit_author_name_tokens):
+        if name in author_last_names:
```

Comment:
you need to compare ASCII- and case-normalized tokens. So `Gómez` should match `gomez` (everywhere).